src/components/FiltroProjetos.tsx

```
@@ -0,0 +1,187 @@
+        CursoService.listar().then((cursos) => {
+            const options = cursos
+                .filter((c) => c.id !== undefined)
+                .map((c) => ({
+                    label: c.nome,
+                    value: c.id as number,
+                }))
+            setCursosOptions(options)
+        })
+    }, [])
+
+    const buscarResponsavelPorEmail = async () => {
```

ESLint

[error] Delete `··`

(prettier/prettier)